src/lib/UtapiClient.js

```diff
@@ -169,33 +167,26 @@ export default class UtapiClient {
      */
     _getParamsArr(params) {
         this._checkMetricTypes(params);
+        const props = [];
+        const { byteLength, newByteLength, oldByteLength, numberOfObjects } =
+            params;
+        const metricData = {
```

Comment:
hmm OK, I think you mean `params` has more properties than the ones listed here, right? In other words